.vscode/inventory.yml

```diff
@@ -0,0 +1,21 @@
+---
+anta_inventory:
```

Comment:
do we want these in the repo or should it be to each its own?

Reply:
Fixed by removing files and introducing management via `envFile`